src/content/docs/style-guide/writing-docs/article-templates/branching-docs-guide.mdx

```diff
@@ -0,0 +1,254 @@
+         logo: 'docker'
+       - value: 'vegan'
+         displayName: 'everything but animal products'
+         logo: 'glassfish'
+
+ steps:
+   - filePath: 'src/install/pasta/appInfo.mdx' 
+   - filePath: 'src/install/pasta/boil.mdx'
+   - filePath: 'src/install/pasta/agentConfig.mdx'
+   - filePath: 'src/install/pasta/type/default.mdx'
+     overrides: 
+       - filePath: 'src/install/pasta/type/omni.mdx'
+         selectedOptions: 
+           - optionType: diet
+             options: 
+               - value: 'omni'
+       - filePath: 'src/install/pasta/type/vegan.mdx'
+         selectedOptions: 
+           - optionType: diet
+             options: 
+               - value: 'vegan'
+   - filePath: 'src/install/pasta/sauce.mdx'
+ whatsNextFilePath: 'src/install/pasta/whatsNext.mdx'
+```
+
+## YAML field name details [#yaml-fields]
+
+* agentName: the directory name where everything lives, also tied to the URL (/install/java)
+* title: name of the doc, free text field
+* introFilePath: the intro text for the page
+* agentConfigFilePath: where the agent yml file lives
+* appInfo: where you define the dropdowns
+    * optionType: value used for overrides, used for query parameters, also used in places for component type appInfo.mdx, appInfoConfigOption, giving a name to the variable used elsewhere on the page 
+    * label: name for the dropdown
+    * placeholder: text that's there before you selected anything
+* options
+    * value: the variable for the name for the tile, hooks into the query param for the URL
+    * displayName: the label of the tile
+    * logo: this is an optional path to an image for the tile
+    * icon: use this if you want to use an icon instead of an image (icon name="fe-server”)
+    * recommendedGuided: Boolean, determines whether guided install content displays (optional)
+
+* steps: steps section defines all the steps on the page, the order these are listed determines the order of the steps
+    * filePath: this is the path to a .mdx file (default file content)
+    * overrides: determines if an .mdx file is swapped in 
+        * skip: Boolean, removes content
+        * selectedOptions: this section hooks into the selected values in the app config to replace content
+            * optionType: this connects to the appInfo variable name
+            * options
+                * value: connects to the specific item in a specific dropdown
+
+
+## Branching MDX files [#mdx-overview]
+
+When working with this install framework, there are specific .mdx files to be aware of.
+
+* Intro
+    * Referenced in the YAML by `introFilePath`
+    * This is the content at the top of the page.
+    * Use for intro content.
+* default 
+    * free mdx file
+* appInfoConfigOption 
+    * this is for dropdowns, use whenever you want dropdowns
+* agentConfig 
+    * this is for the APM agent yml config file
+* appInfo 
+    * this step is the one where you display the content (includes recommendedGuided content)
```

Comment:
@paperclypse I'm not clear on the relationship between appInfo and steps. They seem closely related. Could you clarify that?